src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/MetricSourceConfiguration.cs

```diff
@@ -48,7 +48,7 @@ public MetricSourceConfiguration(float metricIntervalSeconds, IEnumerable<Metric
                 throw new ArgumentNullException(nameof(providers));
             }
 
-            RequestRundown = false;
+            RundownKeyword = 0;
 
```

Comment:
In these places where you specified RundownKeyword=0, I'm guessing you also want RetryStrategy=DropKeywordDropRundown?

Reply:
No, in this case we don't have any custom keyword at all. If this scenario doesn't work on the first try. Dropping the non-existent custom keyword won't work either, there is no point retrying at all.

Earlier I would have set this to `DoNotRetry`, but now I can't because `DoNotRetry` will override other merged profiles to `DoNotRetry`, which is not correct, the other profiles might have custom keyword that can potentially be dropped.

Reply:
I think we reached the same conclusion that we are interpreting 'DoNotRetry' differently (see my response to other comment about the asserts)